axelor-account/src/main/java/com/axelor/apps/account/exception/AccountExceptionMessage.java

```diff
@@ -823,12 +823,17 @@ private AccountExceptionMessage() {}
 
   public static final String DEBT_RECOVERY_2 = /*$$(*/ "Reference date undefined." /*)*/;
   public static final String DEBT_RECOVERY_3 = /*$$(*/
-      "Debt recovery method missing for the configuration." /*)*/;
+      "%s : No match for debt recovery method between the method lines in accounting config for company %s and the category of the partner %s (%s) %s" /*)*/;
```

Comment:
It would be better to have : 
"%s : No debt recovery method has been found for the company %s and the category %s %s (Partner %s)"